packages/wonder-blocks-modal/src/components/__tests__/modal-backdrop.test.js

```diff
@@ -98,10 +124,11 @@ describe("ModalBackdrop", () => {
                     footer={<div data-modal-footer />}
                 />
             </ModalBackdrop>,
+            {attachTo: attachElement},
```

Comment:
Was this introduced due to the Jest upgrade? or to fix some false positives?

Reply:
This is enzyme specific. For some reason, certain things involving the DOM (like focus() changing document.activeElement), the component must be actually in the DOM, not just mounted on an element outside of the document.